src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -225,64 +228,49 @@ private LogQuery buildQuery(AdminActivityLogPageData data) {
         return query;
     }
     
+    /**
+     * Selects versions for query.
+     * @param versions
+     * @return
```

Comment:
I don't think the current comment is a complete description of the method though.